core/src/test/scripts/test_data/study_es_0/result_report.html

```diff
@@ -134,15 +134,15 @@ <h4 class="panel-title">General</h4>
                 <td><span class="glyphicon glyphicon-cog" aria-hidden="true"></span><span class="sr-only">Debug</span></td>
                 <td>&ndash;</td>
                 <td>&ndash;</td>
-                <td>Retrieving chromosome lengths from &#39;/cbioportal/core/src/main/scripts/importer/chromosome_sizes.json&#39;</td>
+                <td>Retrieving chromosome lengths from &#39;/home/pnp300/git/cbioportal/core/src/main/scripts/importer/chromosome_sizes.json&#39;</td>
```

Comment:
this is not correct, the test should be generate from inside the docker container to fix the internal path

Reply:
Fixed